src/manifolds/Euclidean.jl

```diff
@@ -132,6 +132,39 @@ embed(::Euclidean, p)
 
 embed!(::Euclidean, q, p) = copyto!(q, p)
 
+@doc raw"""
+    embed(::EmbeddedManifold{𝔽,Euclidean{𝔽},Euclidean{𝔽},ET}, p)
+
+A [`Euclidean`](@ref) `M` manifold can be embedded into a [`Euclidean`](@ref) `N`
+if the length of the array dimension and representation size is elementwise less or equal.
+The remainder is filles with zeros for sure.
+"""
+function embed(M::EmbeddedManifold{𝔽,Euclidean{n,𝔽a},Euclidean{m,𝔽b},ET}, p) where {n,m,𝔽,𝔽a,𝔽b,ET}
+    q = allocate(p, representation_size(M.embedding))
+    embed!(M, q, p)
+    return q
+end
+
+function embed!(
+    ::EmbeddedManifold{𝔽,Euclidean{nL,𝔽a},Euclidean{mL,𝔽b},ET},
+    q,
+    p,
+) where {nL,mL,𝔽,𝔽a,𝔽b,ET}
```

Comment:
I think 𝔽 and 𝔽b should be the same actually.

Reply:
They _were before_, but above you. asked whether the number fields have to be the same. No. You can embed R in C for sure, that's why I changed that.

What do you prefer, your last remark or this ;) ?

Reply:
Sorry, I was too fast here, Fa can be different, but Fb and F should be the same, for sure. Will change that

Reply:
When embedding, for example, R^2 in C^3, you have `EmbeddedManifold{ℂ,Euclidean{Tuple{2},ℝ},Euclidean{Tuple{3},ℂ},ET}`, right? 𝔽a and 𝔽b are different but 𝔽 and 𝔽b are not. Or did I mix something up again?

Reply:
OK, so we have this one clear :slightly_smiling_face: .